telegram/ext/filters.py

```diff
@@ -56,38 +57,34 @@ class BaseFilter(ABC):
 
             >>> Filters.regex(r'(a?x)') | Filters.regex(r'(b?x)')
 
-        With a message.text of `x`, will only ever return the matches for the first filter,
+        With ``message.text == x``, will only ever return the matches for the first filter,
         since the second one is never evaluated.
 
 
-    If you want to create your own filters create a class inheriting from this class and implement
-    a `filter` method that returns a boolean: `True` if the message should be handled, `False`
-    otherwise. Note that the filters work only as class instances, not actual class objects
-    (so remember to initialize your filter classes).
+    If you want to create your own filters create a class inheriting from either
+    :class:`MessageFilter` or :class:`UpdateFilter` and implement a ``filter`` method that
+    returns a boolean: :obj:`True` if the message should be handled, :obj:`False` otherwise.
+    Note that the filters work only as class instances, not actual class objects (so remember to
+    initialize your filter classes).
 
     By default the filters name (what will get printed when converted to a string for display)
     will be the class name. If you want to overwrite this assign a better name to the `name`
     class variable.
 
     Attributes:
         name (:obj:`str`): Name for this filter. Defaults to the type of filter.
-        update_filter (:obj:`bool`): Whether this filter should work on update. If ``False`` it
-            will run the filter on :attr:`update.effective_message``. Default is ``False``.
         data_filter (:obj:`bool`): Whether this filter is a data filter. A data filter should
             return a dict with lists. The dict will be merged with
             :class:`telegram.ext.CallbackContext`'s internal dict in most cases
             (depends on the handler).
     """
 
     name = None
-    update_filter = False
     data_filter = False
 
+    @abstractmethod
     def __call__(self, update):
-        if self.update_filter:
-            return self.filter(update)
-        else:
-            return self.filter(update.effective_message)
+        ...
```

Comment:
Please use `pass` here instead

Reply:
`...` seems to be standart for abstract methods (according to py docs), but I changed it

Reply:
... is just a placeholdder in case you want to put code like "raise NotImplementedError" or "pass" or similar as far as i know.